manager/volume.go

```diff
@@ -571,9 +573,8 @@ func (m *VolumeManager) Activate(volumeName string, frontend string) (v *longhor
 		return nil, fmt.Errorf("invalid frontend %v", frontend)
 	}
 
-	// ListBackupVolumes() will trigger the update for LastBackup
-	_, err = m.ListBackupVolumes()
-	if err != nil {
+	// Trigger update the LastBackup
+	if err := m.ForceSyncBackupVolume(volumeName); err != nil {
 		logrus.Warnf("failed to update LastBackup and backup volume list before activating standby volume %v: %v", volumeName, err)
 	}
```

Comment:
VolumeController should be responsible for sync backup fields of volumes with BackupVolume CRs. No need to do anything here now.